src/components/CreateVmWizard/steps/Networking.js

```diff
@@ -383,20 +384,26 @@ class Networking extends React.Component {
     const vnicList = createVNicProfileList(vnicProfiles, { dataCenterId, cluster })
     const enableCreate = vnicList.length > 0 && Object.keys(this.state.editing).length === 0
 
-    const nicList = nics
+    const nicList = [...nics]
```

Comment:
```suggestion
    const nicList = nics.slice(0)
```

It is interesting to compare the various ways to create a shallow copy of an array.  Looks like slice(0) is the fastest across browsers.  I ran this in FF and Chrome:
  - https://measurethat.net/Benchmarks/Show/9356/0/array-slice-vs-slice0-vs-spread-operator

Reply:
On further thought, the spread operator is ok in this scenario.  @rszwajko is correct, the spread does look better than `.slice(0)`.

Change is not needed.

Reply:
Now it feels like looking better is more than being faster. Could you provide some more explanation why we prefer the spread operator?

Reply:
> being faster

.... in given test scenario on given browser version. Performance tweaks are tricky ;/

>  we prefer

well... I prefer:)

> some more explanation 

IMHO this syntax is very concise(short) and better shows the intention of the the programmer(copy of the whole array). 
As for more objective reasons - first `slice` can be easily switched by mistake with `splice`. 
Second `slice` main purpose is to copy portion of an array - making a copy of the whole array is supported but may be confusing.

Reply:
Okay, Thanks very much for the explanation, Radek! :)